internal/exec/process.go

```diff
@@ -223,24 +225,25 @@ func startProcess(argv0 string, argv []string, attr *ProcAttr) (pid int, handle
 		}
 	}
 
-	si := new(syscall.StartupInfo)
-	si.Cb = uint32(unsafe.Sizeof(*si))
-	si.Flags = syscall.STARTF_USESTDHANDLES
+	si := new(winapi.STARTUPINFOEX)
+	si.StartupInfo.Cb = uint32(unsafe.Sizeof(*si))
+	si.StartupInfo.Flags = syscall.STARTF_USESTDHANDLES
 	if sys.HideWindow {
-		si.Flags |= syscall.STARTF_USESHOWWINDOW
-		si.ShowWindow = syscall.SW_HIDE
+		si.StartupInfo.Flags |= syscall.STARTF_USESHOWWINDOW
+		si.StartupInfo.ShowWindow = syscall.SW_HIDE
 	}
 	si.StdInput = fd[0]
 	si.StdOutput = fd[1]
 	si.StdErr = fd[2]
```

Comment:
Am I missing something? Does this build? Why aren't these values included in the `StartupInfo`?

Reply:
Discussed offline, closing this